packages/pages/src/upgrade/pagesUpdater.ts

```diff
@@ -280,19 +281,31 @@ export const replaceReactComponentsImports = (source: string): boolean => {
 
 /**
  * Removes old fetch import that is no longer used
- * @param root folder that contains package.json and tsconfig.json
+ * @param source the src folder
  */
-export const removeFetchImport = (root: string) => {
+export const removeFetchImport = (source: string) => {
   let hasRemoved = false;
   const project = new Project({
-    tsConfigFilePath: path.resolve(root, "tsconfig.json"),
+    compilerOptions: {
+      jsx: typescript.JsxEmit.ReactJSX,
+      sourceRoot: source,
+    },
   });
+  project.addSourceFilesAtPaths([
+    `${source}/**/*.ts`,
```

Comment:
Does `${source}/**/*.{ts,js,tsx,jsx}` work?

Reply:
Yes, I've tested on the starter repo and verified that the fetch import in static.tsx is removed.